platform/features/layout/src/LayoutController.js

```diff
@@ -79,6 +83,11 @@ define(
                     ],
                     dimensions: self.defaultDimensions()
                 };
+
+                // Store the id so that the newly-dropped object
+                // gets selected during refresh composition
+                self.droppedFrameId = id;
```

Comment:
@psarram It shouldn't being generic or reusable when there's no need for it to be generic or reusable.  YAGNI.